schema/bom-1.6.schema.json

```diff
@@ -3813,6 +3845,678 @@
+                "prime239v1",
+                "prime239v2",
+                "prime239v3",
+                "prime256v1",
+                "c2pnb176w1",
+                "c2pnb163v1",
+                "c2pnb163v2",
+                "c2pnb163v3",
+                "c2pnb208w1",
+                "c2tnb191v3",
+                "c2tnb191v2",
+                "c2tnb191v1",
+                "c2tnb239v3",
+                "c2tnb239v2",
+                "c2tnb239v1",
+                "c2pnb272w1",
+                "c2pnb304w1",
+                "c2pnb368w1",
+                "c2tnb359v1",
+                "c2tnb431r1",
+                "c2onb191v4",
+                "c2onb191v5",
+                "c2onb239v4",
+                "c2onb239v5",
+                "ansit163k1",
+                "ansit163r1",
+                "ansit163r2",
+                "ansit193r1",
+                "ansit193r2",
+                "ansit233k1",
+                "ansit233r1",
+                "ansit239k1",
+                "ansit283k1",
+                "ansit283r1",
+                "ansit409k1",
+                "ansit409r1",
+                "ansit571k1",
+                "ansit571r1",
+                "ansip160k1",
+                "ansip160r1",
+                "ansip160r2",
+                "ansip192k1",
+                "ansip224k1",
+                "ansip224r1",
+                "ansip256k1",
+                "ansip384r1",
+                "ansip521r1",
+                "FRP256v1",
+                "bn158",
+                "bn190",
+                "bn222",
+                "bn254",
+                "bn286",
+                "bn318",
+                "bn350",
+                "bn382",
+                "bn414",
+                "bn446",
+                "bn478",
+                "bn510",
+                "bn542",
+                "bn574",
+                "bn606",
+                "bn638",
+                "P-192",
+                "P-224",
+                "P-256",
+                "P-384",
+                "P-521",
+                "K-163",
+                "B-163",
+                "K-233",
+                "B-233",
+                "K-283",
+                "B-283",
+                "K-409",
+                "B-409",
+                "K-571",
+                "B-571",
+                "FourQ",
+                "wap-wsg-idm-ecid-wtls1",
+                "wap-wsg-idm-ecid-wtls3",
+                "wap-wsg-idm-ecid-wtls4",
+                "wap-wsg-idm-ecid-wtls5",
+                "wap-wsg-idm-ecid-wtls6",
+                "wap-wsg-idm-ecid-wtls7",
+                "wap-wsg-idm-ecid-wtls8",
+                "wap-wsg-idm-ecid-wtls9",
+                "other",
+                "unknown"
+              ]
+            },
+            "executionEnvironment": {
+              "type": "string",
+              "title": "Execution Environment",
+              "description": "The target and execution environment in which the algorithm is implemented in.\n\n* TODO Basil",
```

Comment:
:exclamation: TODO

Reply:
@bhess you're mentined in an open TODO here.

Reply:
I'll address the TODOs with my name in it this week.

Reply:
I opened PR #350 which addresses the TODOs.

Reply:
#350 was merged